calculator.py

```diff
@@ -1,2 +1,65 @@
+def checkforerror(input):
+    inputlist = [char for char in input]  # Segments out each charcter in a list
+    for item in inputlist:
+        if item == '0' or item == '1':  # Makes sure it is only 0 or 1
+            continue
+        else:
+            return False # If it is anything else it sends a false flag
```

Comment:
This could be simplified by returning `False` if the item isn't `0` or `1`, like this:
```suggestion
        if item != '0' and item != '1':
            return False
```